gnoland/website/views/funcs.html

```diff
@@ -1,59 +1,178 @@
 {{ define "header_buttons" }}
 <div id="header_buttons">
-  <a href="https://github.com/gnolang/gno"><img src="/static/img/github-mark-32px.png" alt="Github"/></a>
+  <a href="https://github.com/gnolang/gno"
+    ><img src="/static/img/github-mark-32px.png" alt="Github"
+  /></a>
+  <a href="#" title="Toggle dark mode" id="theme-toggle">
+    <!--
+      Copyright (c) 2013-2017 Cole Bemis
+      https://feathericons.com/
+    -->
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="32"
+      height="32"
+      viewBox="0 0 24 24"
+      fill="none"
+      stroke="currentColor"
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+      id="theme-toggle-sun"
+    >
+      <circle cx="12" cy="12" r="5"></circle>
+      <line x1="12" y1="1" x2="12" y2="3"></line>
+      <line x1="12" y1="21" x2="12" y2="23"></line>
+      <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
+      <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
+      <line x1="1" y1="12" x2="3" y2="12"></line>
+      <line x1="21" y1="12" x2="23" y2="12"></line>
+      <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
+      <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
+    </svg>
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="32"
+      height="32"
+      viewBox="0 0 24 24"
+      fill="none"
+      stroke="currentColor"
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+      id="theme-toggle-moon"
+    >
+      <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
+    </svg>
+  </a>
 </div>
-{{ end }}
-
-{{define "html_head"}}
-  <meta name="viewport" content="width=device-width,initial-scale=1" />
-  <link rel="stylesheet" href="/static/css/app.css"/>
-  <link rel="stylesheet" href="/static/css/normalize.css"/>
-  <noscript>
-   	<style type="text/css">
-      #source { display:block; }
-   	</style>
-  </noscript>
-{{ end }}
-
-{{ define "header_logo" }}<a id="logo" href="/"><img src="/static/img/logo.png" alt="Gno.land" title="Gno.land" height="38" /> </a>{{ end }}
+{{ end }} {{define "html_head" }}
+<meta name="viewport" content="width=device-width,initial-scale=1" />
+<link rel="stylesheet" href="/static/css/app.css" />
+<link rel="stylesheet" href="/static/css/normalize.css" />
+<noscript>
+  <style type="text/css">
+    #source {
+      display: block;
+    }
+    #theme-toggle {
+      display: none;
+    }
+  </style>
+</noscript>
+<script>
+  (function () {
+    var storageKey = "website_theme";
```

Comment:
You can make this a constant since you don't modify it

Reply:
Yeah here I was just keeping consistent with the fact that, as far as I've seen, the other JavaScript we have is still pre-ES5 JavaScript (ie. `var` and no `const` or `let`, no arrow functions, no template literals). We could drop the requirement and change the existing JavaScript to at least have const and let instead of var (giving us proper scoping instead of var's "crazy" scoping).